pkgs/by-name/zo/zoom-us/package.nix

```diff
@@ -115,7 +115,7 @@ let
 
   binPath = lib.makeBinPath ([ coreutils glib.dev pciutils procps util-linux ] ++ lib.optional pulseaudioSupport pulseaudio);
 in
-stdenv.mkDerivation rec {
+stdenv.mkDerivation {
   pname = "zoom";
   version = versions.${system} or throwSystem;
```

Comment:
is this `throwSystem` logic necessary given `meta.platforms` already limits it?

Reply:
I wouldn't know, this was already here

Reply:
But I would guess it's needed, in case the `meta.platforms` value and the available packages for each system get out of sync...